### PR TITLE
[docs]: Fix documentation for Tags Component

### PR DESCRIPTION
**Pull request by Vaishnavi-Raykar**
I made formatting updates as well as content updates in give 2 files to solve the issue #10897

### CODE REVIEW

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]

File: docs/docs/widgets/tags.md [hunk 1]
```diff
@@ -25,54 +25,54 @@ It can be used to set array of tags. It must be an array of objects like this:
...
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Layout
 
 | <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `fx` to set the value `{{true}}` or `{{false}}` |
```
[the remark this answers is not in the document]
Comment (Vaishnavi-Raykar) [reply docs/docs/widgets/tags.md]: @RudraPratik30

File: docs/docs/widgets/tags.md [hunk 2]
```diff
@@ -25,54 +25,54 @@ It can be used to set array of tags. It must be an array of objects like this:
...
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Layout
 
 | <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}` |
```
Comment (RudraPratik30) [inline docs/docs/widgets/tags.md:65]: Hey, can you just add the full stops at the end? In the expected value column, probably the last change needed.
Please make the changes in both versions.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]

**Review by RudraPratik30 — DISMISSED** [review]
Hey @Vaishnavi-Raykar approving the PR from my end and forwarding it to the team for final review.
Thanks for contributing!
Your PR will be merged by the end of this week.

**Comment by RudraPratik30** [comment]
Hey @Vaishnavi-Raykar can you please sign the [Contributor License Agreement](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=10904) before we can accept your contribution.

File: docs/versioned_docs/version-2.50.0-LTS/widgets/tags.md [hunk 3]
```diff
@@ -25,54 +25,54 @@ It can be used to set array of tags. It must be an array of objects like this:
 }}
 ```
 
-Each object should contain a title, color code of a particular tag, and also a text color.
+Each object should contain a different title in different background colors like success in green color, info in blue color, warning in orange color and danger in red color while a text color is white to all objects.
```
Comment (amanregu) [inline docs/versioned_docs/version-2.50.0-LTS/widgets/tags.md:29]: The usage of the word `different` here makes it feel that you cannot have identical tags or identical titles or colours, which is not the actual behaviour.

Comment (RudraPratik30) [reply docs/versioned_docs/version-2.50.0-LTS/widgets/tags.md:29]: Hey @Vaishnavi-Raykar, please make the above change in both versions.
Thanks!

Comment (Vaishnavi-Raykar) [reply docs/versioned_docs/version-2.50.0-LTS/widgets/tags.md:29]: @amanregu @RudraPratik30 Done Sir

**Comment by RudraPratik30** [comment]
Hey @Vaishnavi-Raykar, thanks for contributing!
You can expect your PR to be merged by the end of this week.

**Review by RudraPratik30 — APPROVED** [review]

**Comment by Vaishnavi-Raykar** [comment]
@amanregu, Can u please review this PR?

**Comment by RudraPratik30** [comment]
Hey @Vaishnavi-Raykar, we are sorry for the delay. Your PR is under review and you can expect to hear an update on it soon.
Thanks for the patience!

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me.

**Pull request merged**